File: app/src/main/java/com/kushal/ftpImageLoading/glideFTP/FTPModel.java
package com.kushal.ftpImageLoading.glideFTP;

public class FTPModel {

    String imageFile;

    public FTPModel(String imageFile) {
        this.imageFile = imageFile;
    }

    @Override
    public int hashCode() {
        return imageFile.hashCode();
    }

    @Override
    public boolean equals(Object obj) {
        return imageFile.equals(((FTPModel) obj).imageFile);
    }

    public String getImagePath() {
        return imageFile;
    }

    public void setImagePath(String imageFile) {
        this.imageFile = imageFile;
    }
}

File: app/src/main/java/com/kushal/ftpImageLoading/glideFTP/FTPModel.kt
package com.kushal.ftpImageLoading.glideFTP

class FTPModel(var imagePath: String) {

    override fun hashCode(): Int {
        return imagePath.hashCode()
    }

    override fun equals(obj: Any?): Boolean {
        return imagePath == (obj as FTPModel).imagePath
    }
}